src/pages/task/TaskTabs.tsx

```diff
@@ -10,7 +10,9 @@ import { getTaskRoute } from "constants/routes";
 import { GetTaskQuery } from "gql/generated/types";
 import { usePrevious } from "hooks";
 import { useBuildBaronVariables } from "hooks/useBuildBaronVariables";
-import { TaskTab } from "types/task";
+import { useUpdateURLQueryParams } from "hooks/useUpdateURLQueryParams";
+import { toSortString } from "pages/patch/patchTabs/util";
```

Comment:
Can you please move this to a generic util folder. If we need something from another page it should be moved to a more generic folder.